conanfile.py

```diff
@@ -0,0 +1,20 @@
+from conans import ConanFile, tools
+import os
+
+class LibwebrtcConan(ConanFile):
+  name = "libwebrtc"
+  settings = "os", "arch"
+  description = "Libwebrtc"
+  url = "https://github.com/Mersive-Technologies/libwebrtc"
+  license = "Apache License 2.0" # TODO: Perhaps work out the legal ramifications later.
+  author = "Mersive Technologies"
+  topics = None
+  exports = "version.txt"
+
+  def init(self):
+    with open(os.path.join(self.recipe_folder, "version.txt"), "r") as file:
```

Comment:
Maybe I'm missing something... but I don't see self.recipe_folder specified in this class.

Reply:
You're just missing the base knowledge about conanfile.py and the system. Not your fault -- it's all new! 
There's a base conan class we derive from, that implements a bunch of stuff. You can read about `self.recipe_folder` here: https://docs.conan.io/en/latest/reference/conanfile/attributes.html#recipe-folder

Reply:
Ah, makes sense!  Thanks!